src/derivatives/05-diffusion-derivatives.md

```diff
@@ -41,83 +41,81 @@ which has its own unique requirements with respect to:
+my_tensors/
+    sub-01/
         dwi/
-            <source_keywords>[_space-<space>]_model-<label>_param-<param1>_model.nii[.gz]
-            <source_keywords>[_space-<space>]_model-<label>_param-<param1>_model.json
-            <source_keywords>[_space-<space>]_model-<label>_param-<param2>_model.nii[.gz]
-            <source_keywords>[_space-<space>]_model-<label>_param-<param2>_model.json
-            <source_keywords>[_space-<space>]_model-<label>_model.json
-
-            [<source_keywords>[_space-<space>]_model-<label>_param-<parama>_mdp.nii[.gz]]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<parama>_mdp.json]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<paramb>_mdp.nii[.gz]]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<paramb>_mdp.json]
-```
+            sub-01_model-DTI_param_Sxx_dwimap.nii.gz
+            sub-01_model-DTI_param_Sxy_dwimap.nii.gz
+            sub-01_model-DTI_param_Sxz_dwimap.nii.gz
+            sub-01_model-DTI_param_Syy_dwimap.nii.gz
+            sub-01_model-DTI_param_Syz_dwimap.nii.gz
+            sub-01_model-DTI_param_Szz_dwimap.nii.gz
+            sub-01_model-DTI_dwimap.json
+            sub-01_model-DTI_param-FA_dwimap.nii.gz
+            sub-01_model-DTI_param-MD_dwimap.nii.gz
 
--   Files "`<source_keywords>[_space-<space>]_model-<label>_param-<param*>_model.nii[.gz]`"
-    provide data corresponding to the various requisite [model fit parameters](#paramdef-fit).
+```
 
-    In cases where *all* [model fit parameters](#paramdef-fit) are contained within a single image
-    file, entity "`_param-`" MUST NOT be included; e.g.:
-        ```Text
-        <pipeline_name>/
-            sub-<participant_label>/
-                dwi/
-                    <source_keywords>[_space-<space>]_model-<label>_model.nii[.gz]
-                    <source_keywords>[_space-<space>]_model-<label>_model.json
-        ```
+In cases where a set of different parameters are contained within a single image
+file, entity "`_param-`" MUST NOT be included; e.g.:
+    ```Text
+    <pipeline_name>/
+        sub-<participant_label>/
+            dwi/
+                <source_keywords>[_space-<space>]_model-<label>_dwimap.nii[.gz]
+                <source_keywords>[_space-<space>]_model-<label>_dwimap.json
+    ```
 
--   File "`<source_keywords>[_space-<space>]_model-<label>_model.json`"
-    provides basic model information and [input parameters](#paramdef-input).
+Example:
 
--   OPTIONAL images "`<source_keywords>[_space-<space>]_model-<label>_param-<parama>_mdp.nii[.gz]`"
-    may be defined, in order to provide additional [model-derived parameters](#paramdef-derived).
+```Text
+my_tensors/
+    sub-01/
+        dwi/
+            sub-01_model-DTI_dwimap.nii.gz # Contains the six diffusion tensor components
+            sub-01_model-DTI_dwimap.json
+            sub-01_model-DTI_param-FA_dwimap.nii.gz
+            sub-01_model-DTI_param-MD_dwimap.nii.gz
```

Comment:
This is implicitly dependent on the very distinction between model fit and model-derived parameters that it seeks to remove.

Here there are three `model-DTI` `dwimaps`:

1. "`FA`"
2. "`MD`"
3. unnamed.

If one knows that "the tensor" is the six coefficients of a symmetric rank-2 tensor that are fit to the image data, and parameters FA and MD are calculated from that, then promoting the former to be an unnamed parameter makes sense. But as soon as everything becomes a `dwimap`, everything needs to be named.

Reply:
As per #74 & other comments here, my intuition is to not use `param-tensor`. The tensor is a data representation by which anisotropic information on S2 can be encoded. Also there's other discrete mixture models that may yield multiple tensors. Ultimately the quantitative parameter that these data are encoding is diffusion coefficient; it's just using a tensor encoding to represent the anisotropy of such.